CHANGELOG.rst

```diff
@@ -7,6 +7,16 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.6.5 (2024-03-28)
+--------------------------
+
+* Added: ``audb.Dependencies.__eq__()``
+  to compare two dependency tables
+* Fixed: let ``audb.available()``
+  skip non-complete datasets
+  instead of raising an error
```

Comment:
With respect to the discussion in the issue: should one not say raise a `FileNotFoundError` when the db directory is not there - specifying the reason for incompleteness?

Reply:
I would not raise an error in `audb.available()`, because then you cannot run it until the dataset is fixed, which can take weeks. The task of `audb.available()` is to find all complete datasets, that a user can load. I would say it is safe to simply ignore all other datasets.
If we think we need a function to find broken datasets, we should add it.